nbconvert/nbconvertapp.py

```diff
@@ -56,6 +56,7 @@ def validate(self, obj, value):
     'template' : 'TemplateExporter.template_name',
     'template-file' : 'TemplateExporter.template_file',
     'theme' : 'HTMLExporter.theme',
+    'embed-images' : 'HTMLExporter.embed_images',
```

Comment:
perhaps make it a `flag`, if it defaults to `False`?

Reply:
thanks for the review! Makes total sense, I did the change